### ISSUE TITLE
less chokes on extended CSS selectors

### ISSUE THREAD

**Issue by mustangostang** [less/less.ruby#22]
Less 0.8.10 can't parse the following simple CSS selector (and any selector with brackets inside, I guess):

```
input[type="text"] { border: 1px solid black }
```

The error message is the following:

```
!! errors were found in the .less file!
line 1: odd number list for Hash
{input[type='text']" "=>{ "border"=>"1px solid black",},}
                    ^
line 1: syntax error, unexpected tSTRING_BEG, expecting '}'
{input[type='text']" "=>{ "border"=>"1px solid black",},}
                    ^
line 1: syntax error, unexpected tASSOC, expecting $end
{input[type='text']" "=>{ "border"=>"1px solid black",},}
                        ^
```

**Comment by cloudhead** [comment]
This'll be fixed in 0.8.11, sometime today. thanks.

**Comment by cloudhead** [comment]
Closed by 175c9b5a41dcaad0a82037d50f9e3ff31692a2da

**Comment by cloudhead** [comment]
0.8.11 is out!
